From 3b23099f8a9c1a93f51e6b4285c9e1435bc112f9 Mon Sep 17 00:00:00 2001
From: Tom Marshall <tdm.code@gmail.com>
Date: Tue, 26 Jun 2018 18:42:20 +0200
Subject: [PATCH 2/2] toybox: Add install to symlinks

Because apparently "mkdir -p" is too pedestrian for some packages.

install is already built/configured (it is part of the cp code), only
the symlink is missing.

Change-Id: Iac4abaeabe72c9c77ca3b99143273d540a406dfa
---
 Android.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Android.mk b/Android.mk
index 116fc48b..8ab3c673 100644
--- a/Android.mk
+++ b/Android.mk
@@ -293,6 +293,7 @@ ALL_TOOLS := \
     ifconfig \
     inotifyd \
     insmod \
+    install \
     ionice \
     iorenice \
     kill \
-- 
2.17.1

